src/riak_kv_app.erl

```diff
@@ -212,6 +212,11 @@ start(_Type, _StartArgs) ->
                                            riak_ql_ddl_compiler:get_compiler_capabilities(),
                                            lists:last(riak_ql_ddl_compiler:get_compiler_capabilities())),
 
+            riak_core_capability:register({riak_kv, sql_select_version},
```

Comment:
Other versioned things like 2i indices and the object format use v1, v2 and v3 for versions not 1, 2, 3 - we should stick with that convention

v1, v2, and v3 behave with >, < and = like 1, 2, 3 - the list generator is just a big clunkier...